models/User.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+const Habit = require('./Habit');
+
+const { Schema } = mongoose;
+
+// Create User schema
+const userSchema = new Schema({
+	username: {
+		type: String,
+		required: true,
+	},
+	email: {
+		type: String,
+		required: true,
+	},
+	password: {
+		type: String,
+		required: true,
+	},
+	// to record the date when the user registered
+	date: {
```

Comment:
Maybe rename this to something like `dateJoined` to be more clear what's this value about?

Reply:
✔️